src/Tooltip/mobile-view.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react'
+import ReactMarkdown from 'react-markdown'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+class MobileView extends Component {
+  state = {
+    visible: false
+  }
+
+  openTooltip = e => {
+    e.stopPropagation()
+    this.setState({
+      visible: true
+    })
+  }
+
+  closeTooltip = e => {
+    e.stopPropagation()
+    this.setState({
+      visible: false
+    })
+  }
+
+  render() {
+    return (
+      <>
+        <HighlightedText onClick={this.openTooltip}>
+          {this.props.text}
+        </HighlightedText>
+        {this.state.visible && (
```

Comment:
this will be inserting `div` into DOM after the term, right? It breaks the layout sometimes in case like this `term.`. Browsers stop treating `word.` as a single non-wrappable entity and treat `word` and `.` as two separate items and in certain cases it leads to re-layout when `word` moves to the previous line and `.` stays on the same when tooltip `div` is visible. The workaround is to move hidden div _before_ the term. Not ideal, but will solve 99% of the problems.

Reply:
No, it won't. I have made a `div` tag with ID - `modal-root` which is at the same level as `bodybag`. It inserts the modal inside it. I have followed [React Portals](https://reactjs.org/docs/portals.html) tutorial for this.

Reply:
👍